library/ceph_pool.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+DOCUMENTATION = """
+---
+author: Michael Sambol
+module: ceph_pool
+short_description: Creates ceph pool and ensures correct pg count
+description:
+  There are three possible outcomes:
+    1/ Create a new pool if it doesn't exist
+    2/ Set the pool's pg count to correct number
+    3/ Nothing: pool is created and pg count is correct
+options:
+  pool_name:
+    description:
+      - The pool in question: create it or ensure correct pg count
+    required: true
+  constant:
+    description:
+      - A constant multiplier used in calculating pg count
+    required: true
+  copies:
+    description:
+      - The number of copies the cluster keeps
+    required: true
```

Comment:
yeah, it's a pretty "custom" module which I don't really like but I thought was the best way to do what we needed. 

I could add that check it:
```
root@qint-lon02-c4:~# ceph osd pool get default size
size: 3
```